pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.sonatype.oss</groupId>
         <artifactId>oss-parent</artifactId>
```

Comment:
Sonatype parent is deprecated (spilt into two suggestions because Github doesn't support deleted lines for suggestions)
```suggestion
```